### ISSUE TITLE
Docs failing

### ISSUE THREAD

**Issue by LukasNickel** [cta-observatory/ctapipe#1802]
In the most recent PRs #1799  and #1801 the docs action is failing due to a missing `pip` installation for python 3.8.
```
opt/hostedtoolcache/Python/3.8.12/x64/bin/python: No module named pip
```
Does anyone know what changed to cause this?

**Comment by HealthyPear** [comment]
weird...we are installing it
https://github.com/cta-observatory/ctapipe/blob/8851e1214409eac4564996cc0f4b76dfe05cf9cf/.github/workflows/ci.yml#L76

**Comment by maxnoe** [comment]
Should be fixed in #1804
